service_crategen/services.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
Perhaps. If it just scans the botocore directory, we lose control over which services to generate. This is sort of based off [this config](https://github.com/aws/aws-sdk-net/blob/master/generator/ServiceModels/_sdk-versions.json) in the .Net SDK. I think it could be expanded in the future though. One thing I'd probably like to do is decouple the versions from each other.